packages/app/src/app/pages/Pro/components/SubscriptionCard.tsx

```diff
@@ -0,0 +1,155 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import { Feature } from 'app/constants';
+import { Text, Stack, Element, Badge } from '@codesandbox/components';
+
+const StyledCard = styled.div<{ isHighlighted?: boolean }>`
+  min-width: 222px;
+  flex-grow: 0;
+  padding: 24px;
+  font-size: 13px;
+
+  ${props =>
+    props.isHighlighted
+      ? `
+    background-color: #FFFFFF;
+    color: #0E0E0E;
+    `
+      : `
+    background-color: #252525;
+    color: #999999;
+  `}
+
+  @media (min-width: 887px) {
+    width: 320px;
+  }
```

Comment:
I couldn't find mobile designs for this but would it be better to have a width set for smaller breakpoints? Tablet could use some wrapping to keep things inline if it has space and a bit of spacing around the cards.

![qlfygh-3000 preview csb app_pro](https://user-images.githubusercontent.com/24959348/204948520-f4753eac-f2a7-441e-b0f3-7a93fd5c3b51.png)